src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -136,7 +151,44 @@ def get(self, svc_id):
         daemon['rgw_metadata'] = metadata
         daemon['rgw_status'] = status
         return daemon
-
+    
+    @RESTController.Collection(method='GET', path='/multisite')
+    @allow_empty_body
+    def get_multisite_info(self):
```

Comment:
get is pretty much self explained through the API method='GET'
```suggestion
    def multisite_info(self):
```

Reply:
Done.